### PR TITLE
Gate zone

### PR DESCRIPTION
**Pull request by young-seoulrobotics**
- Add `LABEL_GATE` to `LabelType`
- ~~Fix the duplicated enum value from algo node status~~

### CODE REVIEW

File: type.proto [hunk 1]
```diff
@@ -14,7 +14,8 @@ enum LabelType {
   LABEL_CYCLIST = 3;
   LABEL_MISC = 4;
   LABEL_GROUND = 5;
-  LABEL_MAX = 6;
+  LABEL_GATE = 6;
+  LABEL_MAX = 7;
```
Comment (young-seoulrobotics) [inline type.proto:18]: Do we need this value? Protobuf automatically generates the max enum value like ArraySize so it's redundant in my opinion.

File: type.proto [hunk 2]
```diff
@@ -14,7 +14,8 @@ enum LabelType {
...
 }
 
 enum ZoneType {
```
Comment (young-seoulrobotics) [inline type.proto:21]: I'm not sure if I have to add `ZONE_TYPE_GATE` to this enum. Any thoughts?

Comment (KangJH) [reply type.proto:21]: what is difference `LABEL_GATE` and `ZONE_TYPE_GATE` ?

**Review by KangJH — APPROVED** [review]

File: type.proto (hunk 2 above)
Comment (young-seoulrobotics) [reply type.proto:21]: It needs some more background information, so let me explain it. I recently added a zone type called `GateZone`. This has to be set tightly around the gate, and objects fully inside the gate are classified as `Gate`. `ZONE_TYPE_GATE` might be required if we want to create such a zone through REST API.

Comment (KangJH) [reply type.proto:21]: ok, if the zone type is used in all product,  then we need to add it.

Comment (KangJH) [reply type.proto:21]: @minhup I'm pretty sure gate zone is for all products, am I correct?

Comment (minhup) [reply type.proto:21]: @KangJH No, it's for only sensr S for now.

**Pull request merged**